libfuse/fs.go

```diff
@@ -199,18 +199,22 @@ func (f *FS) Root() (fs.Node, error) {
 
 // Statfs implements the fs.FSStatfser interface for FS.
 func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
-	// TODO: Fill in real values for these.
-	var bsize uint32 = 32 * 1024
 	*resp = fuse.StatfsResponse{
-		Blocks:  ^uint64(0) / uint64(bsize),
-		Bfree:   ^uint64(0) / uint64(bsize),
-		Bavail:  ^uint64(0) / uint64(bsize),
-		Files:   0,
-		Ffree:   0,
-		Bsize:   bsize,
+		Bsize:   fuseBlockSize,
 		Namelen: ^uint32(0),
-		Frsize:  0,
+		Frsize:  fuseBlockSize,
 	}
+	status, _, err := f.config.KBFSOps().Status(ctx)
```

Comment:
>  I guess it can be an optimization for a later PR though

OK! I'll add the time-based cache first and see how complicated the hint part is ...

>  how often it calls `statfs`

IIRC, it's fairly often. I'll make sure. But doesn't expiring at 5s mean it's at-most 1 per-5s? Is that bad?